dev/integration_tests/flutter_gallery/lib/demo/shrine/supplemental/product_card.dart

```diff
@@ -35,7 +35,7 @@ class ProductCard extends StatelessWidget {
 
     // The fontSize to use for computing the heuristic UI scaling factor.
     const double defaultFontSize = 14.0;
-    final double containerScaingFactor = MediaQuery.textScalerOf(context).scale(defaultFontSize) / defaultFontSize;
+    final double containerScalingFactor = MediaQuery.textScalerOf(context).scale(defaultFontSize) / defaultFontSize;
```

Comment:
Looks like there are some unrelated text scaling changes

Reply:
Yeah I was trying to smuggle a typo fix. Will do that in a different PR then.